spec/rails_helper.rb

```diff
@@ -50,6 +50,8 @@
 
   config.before(:each) do
     DatabaseCleaner.strategy = :transaction
+    # note, it's useful to temporarily switch this to 'truncation' when attempting to debug database oddities during testing
+    # DatabaseCleaner.strategy = :truncation
```

Comment:
If you add another block like the `before(:each, js: true)` one for `before(:each, truncate: true)` you can tag a spec with `:truncate` to get this behavior without changing the strategy globally (either temporarily for debugging purposes, or because there's a reason it's needed).